test/TestUtilities/SystemUtil.hpp

```diff
@@ -20,20 +20,20 @@ class Angle;
 class String;
 class Time;
 
-std::ostream& operator<<(std::ostream& os, const sf::Angle& angle);
-std::ostream& operator<<(std::ostream& os, const sf::String& string);
-std::ostream& operator<<(std::ostream& os, sf::Time time);
+std::ostream& operator<<(std::ostream& os, const Angle& angle);
+std::ostream& operator<<(std::ostream& os, const String& string);
+std::ostream& operator<<(std::ostream& os, Time time);
 
 template <typename T>
-std::ostream& operator<<(std::ostream& os, const sf::Vector2<T>& vector)
+std::ostream& operator<<(std::ostream& os, const Vector2<T>& vector)
 {
     os << std::fixed << std::setprecision(std::numeric_limits<T>::max_digits10);
     os << "(" << vector.x << ", " << vector.y << ")";
     return os;
```

Comment:
While at it,
```suggestion
    return os << "(" << vector.x << ", " << vector.y << ")";
```

Reply:
Thank you! I would have be so annoyed I missed that before this got merged.